### DISCUSSION TITLE
Rating

### DISCUSSION THREAD

**Discussion by vitorvlopes** [Telefonica/mistica-design#1565] [Proposals]
### Goal

Incorporate the component into the DS Library.

### Context

The component will be a valuable addition to our Design System, as it is used in many types of interfaces, including the "Benefícios", "Assinatura" and "Loja" tabs. Additionally, it is present in the App's own evaluation journey, making it a critical element in our design language. Examples of this component's implementation are attached for reference.

### Impact

If added to the Design System, the component will have a significant impact on creating interfaces for the app's rating journeys. The inclusion of the component in the design library makes creating these screens much easier, faster, and consistent, allowing the development team to focus on other critical areas of the app. This also helps increase the efficiency of the development process, reducing the time needed to design, test, and launch new features. 

### Discussion Questions

- Will this component be global or Vivo Extended?

### Do you have a visual proposal?

https://www.figma.com/file/qL1JgTLQWsSImo5BXjEoKb/Rating---Avalia%C3%A7%C3%A3o?type=design&node-id=625%3A5072&mode=design&t=JAdtUSEbFiBQdohs-1

**Comment by AnaMontes11** [comment]
Hello @vitorvlopes!
We have been carring out review, and we think it makes sense for this to be a global feature. However, to include it, we need to know where this is in your priorities regarding the backlog. Many thanks for your contribution!

**Comment by vitorvlopes** [comment]
Hello, @AnaMontes11. 

Thanks for your message! This component will be implemented in the new app not in MVP. Therefore, its priority is low.
